snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -4,19 +4,16 @@
 
 namespace SequenceExamples

```

Comment:
And possibly get rid of the namespace declarations where possible.

Reply:
Our style would be to use file scoped namespaces for larger samples. Using the global namespace is fine for smaller samples. (I'd call this a larger sample, personally).